pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,9 @@
+# 4.1.6
+
+2020-11-03 (Date of Last Commit)
+
+* Updated the docker for FastqProcessing task to version v0.3.12
```

Comment:
Can you add more of a description here of what exactly changed in this new version?

Reply:
And why it was changed please :)